### PR TITLE
Update design-research.md

### PR DESCRIPTION
**Pull request by paulosacramento**
Added this new research to the page, as suggested on https://github.com/BitcoinDesign/Guide/issues/840

### CODE REVIEW

File: guide/resources/design-research.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [Bitcoin: Knowledge and Perceptions]
+(https://block.xyz/2022/btc-report.pdf)
```
Comment (GBKS) [inline guide/resources/design-research.md:101]: The link is not rendering correctly because there is a line break between the `]` and the `/`.

**Review by GBKS — CHANGES_REQUESTED** [review]
Thanks for adding this. The link is not rendering correctly, left a comment on that.

A couple of small tips:
- The PR title should describe the change you are making
- Good to include a link to the page on the Netlify preview in the PR description, so reviewers can see your changes with one click

File: guide/resources/design-research.md (hunk 1 above)
Comment (paulosacramento) [reply guide/resources/design-research.md]: Thanks for the tips and for correcting this.

**Review by paulosacramento — COMMENTED** [review]
There you go.

**Review by GBKS — APPROVED** [review]

**Comment by GBKS** [comment]
Thanks for the quick update. Looks great now. Will get this merged.

**Pull request merged**